src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  padding: 23px 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  position: relative;
+}
+
+.nav__link:hover {
+  color: #000;
+  text-decoration: underline #00acdc solid 2px;
+}
+
+[data-qa="hover"] {
+  color: #00acdc;
+}
+
+[data-qa="hover"]:hover {
+  color: #00acdc;
+  text-decoration: underline #00acdc solid 2px;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
make variables for all colors in `:root` selector
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties